pyron-plugin/src/main/scala/com/cloudentity/pyron/plugin/util/value/PatternUtil.scala

```diff
@@ -0,0 +1,70 @@
+package com.cloudentity.pyron.plugin.util.value
+
+import scala.annotation.tailrec
+import scala.util.matching.Regex
+
+object PatternUtil {
+
+  def safePatternAndParams(pattern: String): (Regex, List[String]) = {
```

Comment:
Do we have tests for this method? Could you please add scaladoc explaining what it does?

Reply:
'Safe' here means, that we escape regex active symbols, such as very commonly used '.' or '?' but also remaining ones to match string literally, while still capturing params with regex. It's a standard set of regex symbols to be escaped. But  testing with some evil input to double-check could be good.

Reply:
added some tests, but might be better to switch to the implementation already used by rewrites